crates/bitwarden-exporters/src/cxf/identity.rs

```diff
@@ -0,0 +1,696 @@
+use bitwarden_vault::FieldType;
+use credential_exchange_format::{
+    AddressCredential, DriversLicenseCredential, EditableField, EditableFieldCountryCode,
+    EditableFieldDate, EditableFieldString, IdentityDocumentCredential, PassportCredential,
+    PersonNameCredential,
+};
+
+use crate::{Field, Identity};
+
+/// Helper trait to extract value from various EditableField types
+trait ExtractValue {
+    fn extract_value(&self) -> String;
+}
+
+impl ExtractValue for EditableField<EditableFieldString> {
+    fn extract_value(&self) -> String {
+        self.value.0.clone()
+    }
+}
+
+impl ExtractValue for EditableField<EditableFieldCountryCode> {
+    fn extract_value(&self) -> String {
+        self.value.0.clone()
+    }
+}
+
+impl ExtractValue for EditableField<EditableFieldDate> {
+    fn extract_value(&self) -> String {
+        self.value.0.to_string()
+    }
+}
+
+/// Generic helper function to create a custom field from any EditableField type
+fn create_custom_field<T: ExtractValue>(
+    editable_field: Option<&T>,
+    field_name: &str,
+) -> Option<Field> {
+    editable_field.map(|field| Field {
+        name: Some(field_name.to_string()),
+        value: Some(field.extract_value()),
+        r#type: FieldType::Text as u8,
+        linked_id: None,
+    })
+}
```

Comment:
@Hinton You requested this function in another PR, would it be OK to dissmiss that review and refactor using this method? (will move it in said refactor PR)